core/modules/layout/tests/layout.test

```diff
@@ -1981,6 +1981,8 @@ class LayoutBlockTest extends BackdropWebTestCase {
     $hero_title_1 = 'Wonderwoman';
     $hero_content_1 = 'Youth is your greatest weapon, your greatest tool.';
     $hero_title_2 = 'Black Widow';
+    // "isnt" is purposefully missing an apostrophe as it was causing tests to
+    // fail.
```

Comment:
Then why not change this to "This is not gonna end well"?

Reply:
I feel like this might be a quote from the aforementioned character, so didn't want to go changing the actual words...